ProcessLib/BoundaryCondition/BoundaryCondition.cpp

```diff
@@ -13,6 +13,23 @@
 #include "BoundaryConditionConfig.h"
 #include "UniformDirichletBoundaryCondition.h"
 #include "UniformNeumannBoundaryCondition.h"
+#include "UniformRobinBoundaryCondition.h"
+
+std::vector<MeshLib::Element*> getClonedElements(
```

Comment:
Please add `static`. Clang will otherwise comply about a function definition w/o preceding declaration or absence of static specifier.